src/hb-ot-var-mvar-table.hh

```diff
@@ -86,6 +86,9 @@ struct MVAR
     return (this+varStore).get_delta (record->varIdx, coords, coord_count);
   }
 
+  inline float apply_var (hb_tag_t tag, int coord) const
```

Comment:
Just name it `get_var()`, which wouldn't take `coord`.  Just return a delta that you would add to it yourself.

Reply:
Done